src/modules/75-cf/pages/feature-flags/FeatureFlagsPage.tsx

```diff
@@ -657,16 +660,17 @@ const FeatureFlagsPage: React.FC = () => {
               )
             }}
           />
-        </Container>
-      )}
-
-      {!loading && emptyFeatureFlags && (
-        <Container width="100%" height="100%" flex={{ align: 'center-center' }}>
-          <NoData imageURL={imageURL} message={getString(searchTerm ? 'cf.noResultMatch' : 'cf.noFlag')}>
-            <FlagDialog environment={environmentIdentifier} />
-          </NoData>
-        </Container>
-      )}
+        ) : (
+          <NoFeatureFlags
+            hasFeatureFlags={hasFeatureFlags || false}
```

Comment:
When would `hasFeatureFlags` not be a boolean?
```suggestion
            hasFeatureFlags={hasFeatureFlags}
```